lib/sengoku/game_server.ex

```diff
@@ -12,23 +12,23 @@ defmodule Sengoku.GameServer do
 
   @default_ai_wait_time_ms 100
 
-  def new(%{} = options) do
+  def new(%{} = options, arena \\ false) do
     game_id = Token.new(8)
-    start_link(game_id, options)
+    start_link(game_id, options, arena)
     {:ok, game_id}
   end
 
-  def start_link(game_id, options) do
-    GenServer.start_link(__MODULE__, {game_id, options})
+  def start_link(game_id, options, arena) do
+    GenServer.start_link(__MODULE__, {game_id, options, arena})
   end
 
-  def init({game_id, options}) do
+  def init({game_id, options, arena}) do
     case Registry.register(:game_server_registry, game_id, :ok) do
       {:ok, _pid} -> {:ok, game_id}
       {:error, reason} -> {:error, reason}
     end
 
-    {:ok, Game.initialize_state(game_id, options)}
+    {:ok, Map.put(Game.initialize_state(game_id, options), :arena, arena)}
```

Comment:
I don't like boolean parameters, but couldn't come up with a more elegant solution to skip broadcasting updates and slowing down AI turns when we're in "Arena mode". Maybe callback functions that get passed in?